auth/auth-router.js

```diff
@@ -65,11 +67,16 @@ router.post("/login", (req, res) => {
     .then(user => {
       user = user[0];
       if (user && bcrypt.compareSync(password, user.password)) {
-        const token = getJwtToken(user.email, user.password);
+        // const token = getJwtToken(user.email, user.password);  // DEPRECATED
```

Comment:
Would removing token break the front end since it has not been changed to use session?

Reply:
Thinking about it, yes.. I'm going to pull this back to a draft PR until I get the Front End sorted..